### ISSUE TITLE
Around 50 files have broken encodings on french.

### ISSUE THREAD

**Issue by maxencealluin** [aparrish/gutenberg-dammit#12]
Hello,

For those of you that intend to use french documents in this corpus, know that on the 2647 french books included 49 have broken encoding and all accent letters are removed.
A quick way to find the culprits is to look the book for the letter 'é'.

**Comment by DBarthe** [comment]
Yes this is annoying. Seems that it's possible to fix it by replacing :
* `c282` => `é`
* `c28a` => `è`
* `c288` => `ê`
* `c285` => `à`
* `c287` => `ç`
* `c293` => `ô`
* `c28c` => `î`
* `c296` => `û`
* ...
